wallet/createtx.go

```diff
@@ -1126,13 +1126,20 @@ func (w *Wallet) purchaseTickets(req purchaseTicketRequest) ([]*chainhash.Hash,
 
 			splitOuts = append(splitOuts, wire.NewTxOut(int64(userAmt), pkScript))
 		}
-
 	}
 
+	// get the estimated fee for the tx
+	sizeEstimate := txsizes.EstimateSerializeSize(len(splitOuts), splitOuts, false)
+	feeEstimate := txrules.FeeForSerializeSize(w.RelayFee(), sizeEstimate)
 	txFeeIncrement := req.txFee
```

Comment:
This variable is named miseadingly, and I think that is the cause of this PR, but this is really the fee rate (in DCR/kB of serialized TX), not an increment (as it used to be in very early versions of bitcoin).  txToOutputsInternal handles this and adds the appropriate fee given this fee rate.  I believe the code is already correct (albeit misleading) and this change will introduce bugs with the fee.

Reply:
noted, the cause of the issue with #963 must be the incorrect fee rate passed when purchasing a ticket then. Using the estimated fee/kB (which becomes the estimated fee rate) as check for the passed fee rate and picking the greater of the two should resolve the issue.

Reply:
closing because further discussion with jrick shows the issue with #963 has to do with the fee estimation calls being used there.